pulsar-core/src/kernel.rs

```diff
@@ -0,0 +1,15 @@
+// TODO: use specific configuration key instead ok target_os and target_arch
```

Comment:
Why?

Reply:
to decouple the variant from the target os